.github/workflows/build.yml

```
@@ -431,7 +431,7 @@ jobs:
       - uses: actions/setup-java@v3
         with:
           distribution: temurin
-          # here we intentionally use an older version so that we also verify java 17 compiles to it
+          # here we intentionally use an older version so that we also verify java 19 compiles to it
```

Could omit the version here, so it doesn't need to be updated so often. Just say:

```suggestion
          # intentionally use Java 11 here to ensure that the newer JDK builds for Java 11
```